index.js

```diff
@@ -28,6 +28,9 @@ var colors = {
   brightBlack: '#747369'
 }
 
+var process = require('process') // excluded from browser
+var isNode = !!process.version
```

Comment:
Good point. This is actually better because Electron render processes have both `window` and `require('process')`, so they were getting treated as Node. If we switch this to your suggestion, they will be treated as browsers, which get colors :)